src/components/Footer.tsx

```diff
@@ -213,20 +201,78 @@ const Logo = styled.div`
 const FooterLinkGroups = styled.div`
 	display: flex;
 	flex: 1;
+	width: 100%;
+
+	@media(max-width:768px){
+		flex-wrap: wrap;
+	}
 `;
 
 const FooterLinkGroup = styled.div`
-	flex: 1;
+	// flex: 1;
 	display: flex;
 	flex-direction: column;
-	gap: 2rem;
+	gap: 3rem;
+	width: 50%;
+
 	> h5 {
 		margin: 0;
 		font-size: 1.2rem;
 	}
+	@media(max-width:768px){
+		width: 50%;
+		margin-bottom: 4rem;
+	}
+
 `;
 
 const SocialsContainer = styled.div`
 	display: flex;
 	gap: 0.5rem;
 `;
+
+const LogoInfoWrapper = styled.div`
+	display: flex;
+	flex-rap: wrap;
+	gap: 5rem;
+	flex: 1;
+	margin-top: 4rem;
+
+
+	@media(max-width:768px){
+		flex-direction: column;
+	}
+`
+
+const CopyrightLogoWrapper = styled.div`
+	display: flex;
+	position: relative;
+	justify-content: space-between;
+	align-items: center;
+	// margin-block: 2.5rem;
+	// flex-wrap: wrap;
+	gap: 2rem;
+	margin: 3rem 0 5rem;
+	width: 100%;
+	// padding:;
+
+	@media(min-width: 600px){
+		&:before{
```

Comment:
why remove the `hr` element and replace it with `:before` pseudo element? The `hr` is more semantic in my opinion. Lemme know your reason

Reply:
The hr element wasn't going across the page, while in the figma file the white line was going across the page. I commented the hr element. What do you think?